packages/plugin-connector/src/data-sources/build-sources.js

```diff
@@ -93,14 +93,18 @@ export const buildSources = async (
 			},
 			/**
 			 * @param {string} name
+			 * @returns {boolean} true if query is included in filters
 			 */
 			isFiltered: (name) =>
-				Boolean(filters?.queries?.has(name) || filters?.queries?.has(`${source.name}.${name}`)),
+				Boolean(filters?.queries?.has(name) || filters?.queries?.has(`${source.name}.${name}`)) ||
+				!filters?.queries,
```

Comment:
Doesn't this make the `Boolean` part irrelevant? If `filters?.queries` is true, then it doesn't matter if either call of `filters?.queries?.has`  is true

Reply:
This should mean:

```
table name generically referenced || table name specifically referenced || no filters were specified

```

Reply:
Ah yeah missed the exclamation mark